package.json

```diff
@@ -21,6 +26,11 @@
     "jquery": "^3.2.1"
   },
   "devDependencies": {
-    "live-server": "^1.2.0"
+    "autoprefixer": "^7.1.1",
+    "chokidar-cli": "^1.2.0",
+    "live-server": "^1.2.0",
+    "node-sass-chokidar": "0.0.2",
+    "npm-run-all": "^4.0.2",
```

Comment:
`npm-run-all` Package for running a group of npm scripts from the same group in parallel. Look at our `start` script on line 12 above.